src/server/lib/getGeolocationRegion.ts

```diff
@@ -4,11 +4,37 @@ import { Europe } from './getRegistrationLocation';
 import { CountryCode } from '@guardian/libs';
 
 export const getGeolocationRegion = (req: Request): GeoLocation => {
-	const header = req.headers['x-gu-geolocation'];
+	/**
+	 * Cypress Test START
+	 *
+	 * This code checks if we're running in Cypress
+	 */
+	const runningInCypress = process.env.RUNNING_IN_CYPRESS === 'true';
+	if (runningInCypress) {
+		const cypressMockStateCookie = req.cookies['cypress-mock-state'];
+
+		// check if the cookie value is an expected value
+		const validCode = ['FR', 'GB', 'US', 'AU'].includes(cypressMockStateCookie);
```

Comment:
DRY -> maybe move this into a lib function, or at least somewhere it can be imported in the two places it's being used

Reply:
Don't we say not to refactor if it's being used in two places, only if it's three? ;)

Reply:
I suppose, at the very least it would be useful to have that array as a `const` so we can add/remove elements easily in both places at once.